esmvaltool/recipes/recipe_REF_CRE.yml

```
@@ -0,0 +1,160 @@
+    - bock_lisa
+    - lauer_axel
+  maintainer:
+    - bock_lisa
+
+
+# Note: the following models are just examples
+datasets:
+  - {project: CMIP6, dataset: MPI-ESM1-2-LR, exp: historical, ensemble: r1i1p1f1, grid: gn}
+
+timerange_for_models: &time_period
+  timerange: '2005/2014'  # can be specified, this is just an example
+
+
+preprocessors:
+
+  full_climatology: &full_climatology_diag
+    climate_statistics:
+      period: full
+    regrid:
+      target_grid: 2x2
+      scheme:
+        reference: esmf_regrid.schemes:ESMFAreaWeighted
+
+  zonal_mean:
+    custom_order: true  # makes preprocessor much faster since input for extract_levels is smaller
+    climate_statistics:
+      period: full
+    regrid:
+      scheme:
+        reference: esmf_regrid.schemes:ESMFAreaWeighted
+      target_grid: 2x2
+    zonal_statistics:
+      operator: mean
+
+
+diagnostics:
+
+  plot_lwcre_maps:
+    description: Plot lwcre climatology maps including reference datasets.
+    variables:
+      lwcre:
+        <<: *time_period
+        mip: Amon
+        preprocessor: full_climatology
+        derive: true
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: obs4MIPs, level: L3B, timerange: '2001/2010', tier: 1, reference_for_monitor_diags: true}
+    scripts:
+      plot: &plot_multi_dataset_default
+        script: monitor/multi_datasets.py
+        plot_folder: '{plot_dir}'
+        plot_filename: '{plot_type}_{real_name}_{dataset}_{mip}'
+        plots:
+          map:
+            common_cbar: true
+
+  plot_swcre_maps:
+    description: Plot swcre climatology maps including reference datasets.
+    variables:
+      swcre:
+        <<: *time_period
+        mip: Amon
+        preprocessor: full_climatology
+        derive: true
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: obs4MIPs, level: L3B, tier: 1, timerange: '2001/2010', reference_for_monitor_diags: true}
+    scripts:
+      plot:
+        <<: *plot_multi_dataset_default
+        plots:
+          map:
+            common_cbar: true
+
+  plot_netcre_maps:
+    description: Plot swcre climatology maps including reference datasets.
+    variables:
+      netcre:
+        <<: *time_period
+        mip: Amon
+        preprocessor: full_climatology
+        derive: true
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: obs4MIPs, level: L3B, tier: 1, timerange: '2001/2010', reference_for_monitor_diags: true}
+    scripts:
+      plot:
+        <<: *plot_multi_dataset_default
+        plots:
+          map:
+            common_cbar: true
+
+  plot_lwcre_profiles:
+    description: Plot lwcre zonal mean including reference datasets.
+    variables:
+      lwcre:
+        <<: *time_period
+        mip: Amon
+        preprocessor: zonal_mean
+        derive: true
+        additional_datasets:
+          - {dataset: CERES-EBAF, project: obs4MIPs, level: L3B, tier: 1, timerange: '2001/2010', reference_for_monitor_diags: true}
```

In addition to switching to CERES Ed4.2, we could also show alternative obs datasets here.

```suggestion
          - {dataset: CERES-EBAF, project: OBS, type: sat, version: Ed4.2, tier: 2, reference_for_monitor_diags: true}
          - {dataset: ESACCI-CLOUD, project: OBS, type: sat, version: AVHRR-AMPM-fv3.0, tier: 2}
          - {dataset: ISCCP-FH, project: OBS, type: sat, version: v0, tier: 2}
```